StoneMasonKarel.java

```diff
@@ -13,5 +13,66 @@
 public class StoneMasonKarel extends SuperKarel {
 
 	// You fill in this part
+	public void run(){
```

Comment:
Solution only works for the world that is given. Questions asks you provide a general solution.

Also think in term of decompositions the way we discussed.

What are the major steps that need to be done to solve the problem at higher level?